CHANGELOG.md

```diff
@@ -13,6 +13,34 @@ next release
 
 ### UI Changes
 
+1.43.0 (2023-03-13)
+-------------------
+### Backend Changes
+
+#### New Features
+
+* Support secure OTLP exporter config for hotrod ([@graphaelli](https://github.com/graphaelli) in [#4231](https://github.com/jaegertracing/jaeger/pull/4231))
+
+#### Bug fixes, Minor Improvements
+* 
+* [chore]: Replace pkg/multierror with standard errors.Join ([@ClementRepo](https://github.com/ClementRepo) in [#4293](https://github.com/jaegertracing/jaeger/pull/4293))
+* [chore] remove pkg/multicloser ([@yurishkuro](https://github.com/yurishkuro) in [#4291](https://github.com/jaegertracing/jaeger/pull/4291))
```

Comment:
What do you think about removing the `[chore]` prefixes and Upper case the sentences (same below):
```
* Replace ...
* Remove ...
```

Reply:
The changelog is auto-generated. I can can make manual changes to make it nicer.